### PR TITLE
Passive scan max alerts option

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

**Comment by psiinon** [comment]
Associated help: https://github.com/zaproxy/zap-core-help/pull/210

File: src/org/zaproxy/zap/extension/pscan/PassiveScanParam.java [hunk 1]
```diff
@@ -51,6 +51,7 @@
 
     private static final String SCAN_ONLY_IN_SCOPE_KEY = PASSIVE_SCANS_BASE_KEY + ".scanOnlyInScope";
     private static final String SCAN_FUZZER_MESSAGES_KEY = PASSIVE_SCANS_BASE_KEY + ".scanFuzzerMessages";
+    private static final String MAX_ALERTS = PASSIVE_SCANS_BASE_KEY + ".maxAlerts";
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscan/PassiveScanParam.java]: The key (and related variables/methods/) could more explicit, `maxAlertsPerRule`?

Comment (psiinon) [reply src/org/zaproxy/zap/extension/pscan/PassiveScanParam.java]: Yeah, good suggestion.
Fixed

**Comment by thc202** [comment]
Thank you!

**Pull request merged**